<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateSchuleGroupDialog</class>
 <widget class="QDialog" name="CreateSchuleGroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schule Group Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Course</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboCourse">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>851</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Professors</string>
    </attribute>
    <widget class="QTableView" name="tableTeachers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>811</width>
       <height>511</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Students</string>
    </attribute>
    <widget class="QPushButton" name="pushAddToGroup">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>130</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushRemoveFromGroup">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>180</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushClear">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>270</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableAll">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>341</width>
       <height>511</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QListWidget" name="listGroup">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>331</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Schedule</string>
    </attribute>
    <widget class="QTableView" name="tableSchedule">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>811</width>
       <height>521</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Please after setting the rooms select the cells</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushOK">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
